navbar.php

```diff
@@ -32,7 +32,7 @@
         <!-- <li><a href="#">HELP</a></li> -->
         <li><a href="./contact.php">CONTACT US</a></li>
         <?php
-        if ($_SESSION["email"] != "") {
+        if (isset($_SESSION["email"])) {
```

Comment:
isset and being an empty string are different things. You can check for both, as in the logout file, you have probably set the session variable to an empty string, and not unset it.